src/custom/api/gnosisProtocol/api.ts

```diff
@@ -274,12 +277,23 @@ const UNHANDLED_ORDER_ERROR: ApiErrorObject = {
   description: ApiErrorCodeDetails.UNHANDLED_CREATE_ERROR,
 }
 
-async function _handleQuoteResponse<T = any, P extends QuoteQuery = QuoteQuery>(
+async function _handleQuoteResponse<T = any, P extends MinimumSentryQuoteData = MinimumSentryQuoteData>(
   response: Response,
   params?: P
 ): Promise<T> {
   if (!response.ok) {
-    const errorObj: ApiErrorObject = await response.json()
+    const errorObj: ApiErrorObject = await response.json().catch((error) => {
+      const sentryError = new Error()
```

Comment:
why not just retuning a`ApiErrorObject` in the body of the catch? This way is simpler, right?

Reply:
not exactly this but done